package.json

```diff
@@ -33,7 +33,7 @@
     "ink-docstrap": "^1.3.0",
     "istanbul-instrumenter-loader": "^2.0.0",
     "jasmine-sinon": "^0.3.2",
-    "jquery": "3.6.0",
+    "jquery": "^3.6.0",
```

Comment:
@ewahlquist-sugarcrm I did check with @asaxena-sugarcrm about the version upgrade. That will be done after this is PR merged.
1. Merge ventana PR
2. Publish ventana package through npm which will change the version number 
3. Update mango PR to reflect the new ventana version

@asaxena-sugarcrm please correct me if I'm wrong

Reply:
Yes this is correct. For #2, you will have to make a separate commit to increase the version number before publishing to npm. You will also need to update the ventana version in the Sidecar package.json once published. 

It looks like there is another PR open to update underscore. I think it would be best to work with @odukhnytskyi-sugarcrm on merging so that you only need to bump ventana once instead of per library upgrade. I think CA missed updating momentjs in here, there are dependabot PRs open for it. It would be best to wrap all these upgrades into 1 ventana version bump.